tools/@angular/tsc-wrapped/src/collector.ts

```
@@ -123,7 +123,13 @@ export class MetadataCollector {
       if (classDeclaration.heritageClauses) {
         classDeclaration.heritageClauses.forEach((hc) => {
           if (hc.token === ts.SyntaxKind.ExtendsKeyword && hc.types) {
-            hc.types.forEach(type => result.extends = referenceFrom(type.expression));
+            hc.types.forEach(type => {
```

yes, it was just harder for me to track down. I'll try that.
Do you think it should be a specific exception for extending Error, or can we always ignore errors from 'extends' metadata?